src/sqoare.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Square(props){
+    return(
+      <div>
+        <button className="bt" onClick={()=> props.onClick()}>{props.value}</button>
```

Comment:
@yossefdiy consider using a better class name for this button like 'start-button'